library/aesce.c

```diff
@@ -46,7 +46,7 @@
 
 #include "aesce.h"
 
-#if defined(MBEDTLS_HAVE_ARM64)
+#if defined(MBEDTLS_ARCH_IS_ARM64)
```

Comment:
In `aesni.c` and `padlock.c`, we guard the implementations with `MBEDTLS_XXX_HAVE_CODE`, I think we should do it similarly.

Reply:
It is addressed in #7078 .